ossm/versions.yaml

```diff
@@ -0,0 +1,55 @@
+# This file defines all the Istio versions supported by this operator.
```

Comment:
This file is no longer used. Now it's using pkg/istioversion/versions.ossm.yaml

Reply:
Done